x-pack/plugins/data_visualizer/public/application/index_data_visualizer/search_strategy/requests/get_document_stats.ts

```diff
@@ -4,15 +4,20 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
+// @ts-nocheck // remove
```

Comment:
Are the changes in this file necessary? Since we hopefully copied everything over we need to `aiops` we should aim for leaving data visualizer related files untouched.

Reply:
Nope, shouldn't be - good catch - I'll remove them 👍